devDocs/startupShutdownInstall.md

```diff
@@ -0,0 +1,96 @@
+# NVDA Starting and Exiting outline
+
+## As a user, NVDA can be started by:
+
+1. For an installed copy:
+    1. Ctrl+Alt+N
+    1. Via secure desktop/ease of access
+    1. Via startup

```

Comment:
```suggestion
    1. Automatically by Ease of Access after signing in to Windows
    1. An automatic restart of NVDA due to a handled crash
```

Reply:
>    1. An automatic restart of NVDA due to a handled crash

Should all the ways of restarting NVDA be mentioned here?

Maybe just 

```suggestion
    1. Automatically by Ease of Access after signing in to Windows
    1. An exiting instance of NVDA starting a new process (see shutting down procedures)
```

Reply:
That would be fine with me.